Crash Report 3.3.1: java.lang.ClassNotFoundException ErrorActivity.onCreate

```
Caused by: java.lang.RuntimeException: 

 
  at android.os.Parcel.readSerializable (Parcel.java:2630)

 
  at android.os.Parcel.readValue (Parcel.java:2416)

 
  at android.os.Parcel.readArrayMapInternal (Parcel.java:2732)

 
  at android.os.BaseBundle.unparcel (BaseBundle.java:269)

 
  at android.os.BaseBundle.containsKey (BaseBundle.java:341)

 
  at org.kiwix.kiwixmobile.core.error.ErrorActivity.onCreate (ErrorActivity.java:99)
```

--- a/core/src/main/java/org/kiwix/kiwixmobile/core/error/ErrorActivity.java
+++ b/core/src/main/java/org/kiwix/kiwixmobile/core/error/ErrorActivity.java
@@ -96,7 +96,7 @@ public class ErrorActivity extends BaseActivity {
 
     Bundle extras = callingIntent.getExtras();
     final Throwable exception;
-    if (extras != null && extras.containsKey(EXCEPTION_KEY)) {
+    if (extras != null && safeContains(extras, EXCEPTION_KEY)) {
       exception = (Throwable) extras.getSerializable(EXCEPTION_KEY);
     } else {
       exception = null;
@@ -189,6 +189,14 @@ public class ErrorActivity extends BaseActivity {
     });
 
     restartButton.setOnClickListener(v -> onRestartClicked());
+  }
+
+  private boolean safeContains(Bundle extras, String key) {
+    try {
+      return extras.containsKey(key);
+    }catch (RuntimeException ignore){
+      return false;
+    }
   }
 
   private void onRestartClicked() {

--- a/core/src/main/java/org/kiwix/kiwixmobile/core/error/ErrorActivity.java
+++ b/core/src/main/java/org/kiwix/kiwixmobile/core/error/ErrorActivity.java
@@ -194,7 +194,7 @@ public class ErrorActivity extends BaseActivity {
   private boolean safeContains(Bundle extras, String key) {
     try {
       return extras.containsKey(key);
-    }catch (RuntimeException ignore){
+    } catch (RuntimeException ignore) {
       return false;
     }
   }

